README.md

```diff
@@ -17,3 +17,24 @@ Set your eslint config to:
   "extends": ["@hipo/eslint-config-base"]
 }
 ```
+
+## TypeScript
+
+If you are using TypeScript, to avoid false `import/no-unresolved` errors, you should add `plugin:import/typescript` to your eslint config **(RECOMMENDED)**:
+
+```
+{
+  "extends": ["@hipo/eslint-config-base", "plugin:import/typescript"]
+}
+```
+
+or you can turn off `"import/no-unresolved"` rule in your eslint config:
+
+```
+{
+  rules:{
+    ...
+    "import/no-unresolved": "off"
```

Comment:
```suggestion
    "import/no-unresolved": "off",
    "no-undef": "off"
```

Can you also add a line describing that we need to disable `no-undef` as TSC already catches those and that rule causes problems when we try to use TS only features like enums